xcode/Subconscious/Shared/Library/SQLite3Database.swift

```diff
@@ -127,7 +107,36 @@ final class SQLite3Database {
         public static func bool(_ bool: Bool) -> Self {
             .integer(bool ? 1 : 0)
         }
-
+        
+        /// Create optional text value
+        public static func text(_ string: String?) -> Value {
```

Comment:
Convenience synonyms for optional values that can be type or nil, which is translated to SQL type or NULL.